src/script/lua_api/l_client.cpp

```diff
@@ -169,6 +174,11 @@ int ModApiClient::l_get_node(lua_State *L)
 {
 	// pos
 	v3s16 pos = read_v3s16(L, 1);
+
+	if (getClient(L)->getCSMFlavourLimits() & CSMFlavourLimit::CSM_FL_LOOKUP_NODES) {
+		// TODO
```

Comment:
'get node' is quite simple, a range check may not be intensive, but is not useful as explained below, a line-of-sight check would certainly be too intensive.
Please don't insult.

> removing all parts of CSM like you want because you don't understand the benefits

It's not just me, it's all our most respected server owners and celeron55, all of whom feel very strongly about this.
I do understand there are benefits, and do not dispute them, but i also see the potential harm too, as do others.
Please stop writng ridiculus and insulting things.